moped-database/migrations/1682011649641_create_new_subcomponents/up.sql

```diff
@@ -0,0 +1,44 @@
+INSERT INTO moped_subcomponents (subcomponent_name) values
+    ('Continuous flow intersection'),
+    ('Curb extension'),
+    ('Detection - Bicycle'),
+    ('Vehicle lane reconfiguration'),
+    ('Protected intersection - Full'),
+    ('Protected intersection - Full - Shared use'),
+    ('Protected intersection - Partial'),
+    ('Radius tightening'),
+    ('Slip lane removal'),
+    ('Smart right'),
+    ('Intersection square-up'),
+    ('Transit corner modification'),
+    ('Bicycle accommodations at PHB'),
+    ('Bicycle signal (standard signal + bicycle signal sign)'),
+    ('Bicycle signal face (experiment)'),
+    ('Bicycle signal face (interim approval)'),
+    ('Detection - Vehicle'),
+    ('Leading bicycle interval'),
+    ('Pedestrian signal head and pushbuttons'),
+    ('Protected left turn phase'),
+    ('Bikeway Direction - Two-way'),
+    ('Station Design - Curbside with Bike Lane Bypass'),
+    ('Station Design - Floating with Bike Lane'),
+    ('Station Design - Island with Bike Lane Behind'),
+    ('Station Design - Shared with Bike Lane in Front'),
+    ('Station Design - Step-out with Shared Landing'),
+    ('Station Design - With SUP Behind'),
+    ('Station Design - With SUP in Front'),
+    ('Bus Stop Consolidation'),
+    ('Bus Stop Maintenance'),
+    ('Bus Stop Modification'),
```

Comment:
switches to title case, where the other ones are in sentence case. not sure if intentional